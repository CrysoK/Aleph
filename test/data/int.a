// Pruebas de números enteros

// Carga
println(1)
println(0)
println(1234567890)
println(-0123456789)

// Suma
println(1 + 1)
println(2 + 10 + 132)
println(321 + 4 + 91 + 000)

// Resta
println(1 - 1)
println(21 - 123 - 545)
println(0 - 64 - 5490)

// División
println(8 / 2)
println(9 / 4)
println(130 / 2 / 5)

println('Fin')
