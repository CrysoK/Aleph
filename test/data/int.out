1
0
1234567890
-123456789
2
144
416
0
-647
-5554
4
2
13
Fin
